app/views/layouts/application.slim

```diff
@@ -18,147 +18,146 @@ html
     = render partial: 'application/favicon'
     = render partial: 'shared/twitter_card'
     = content_for(:head)
-  body data-controller=controller_name data-action=action_name
     = Gon::Base.render_data
+  body data-controller=controller_name data-action=action_name
```

Comment:
Do yourself a favor and turn off whitespace changes in the options menu at the top of GitHub when reviewing this file